R/interPlateNorm.R

```diff
@@ -225,7 +225,7 @@ interPlateNorm <- function(data_list,
   if(!is.null(transformReverse)){
     reverses <- which(names(data_list[[i]][,1]) %in% transformReverse) 
     for (i in 1:length(data_list)){
-      data_list[[i]][reverses, ] <- transformReverseMax - data_list[[i]][reverses, ]
+      data_list[[i]][reverses, ] <- transformReverse_scaleFactor^2 / (data_list[[i]][reverses,] + 1)
```

Comment:
why is it power of two? e.g. transformReverse_scaleFactor^2

Reply:
I just fixed this... it was to undo the first scale factor, as data_list has already been scaled by 10^4, but this assumed that the scaleFactor = transformReverse_scaleFactor, which might not always be the case, so I changed it